File: Dataset/Leetcode/train/75/199.cpp
class Solution {
public:
    void XXX(vector<int>& nums) {
        int l=-1,r=nums.size(),i=0;
        while(i<r){
            if(nums[i]<1){
                swap(nums[i++],nums[++l]);
            }else if(nums[i]>1){
                swap(nums[i],nums[--r]);
            }else{
                ++i;
            }
        }
    }
};

File: Dataset/Leetcode/train/56/199.py
 class Solution:
    def XXX(self, nums: List[List[int]]) -> List[List[int]]:
        ans, l, r = [], -float('inf'), -float('inf')
        for i, j in nums.sort() or nums:
            if i <= r:
                r = max(r, j)
            else:
                ans.append([l, r])
                l, r = i, j
        return ans.append([l, r]) or ans.pop(0) and ans

undefined
for (i = 0; i < document.getElementsByTagName("code").length; i++) {     console.log(document.getElementsByTagName("code")[i].innerText); }


